changelogs/CHANGELOG-3.md

```diff
@@ -5023,7 +5023,7 @@ not just made modules using it difficult to deploy.
 Again, this is a _**BETA RELEASE**_, so not everything is working just yet.
 Here are the issues that we already know about. If you run into something
 that isn't on this list,
-[let us know](https://github.com/npm/npm/issues/new)!
+[let us know](https://github.com/npm/cli/issues/new)!
```

Comment:
Ah, looks like this is a broken link also, because `npm/cli` [moved their issues to Discourse](https://blog.npmjs.org/post/175587538995/announcing-npmcommunity).

Maybe the [#support](https://npm.community/c/support) or [#bugs](https://npm.community/c/bugs) categories would be appropriate here.

cc @zkat

Reply:
No need to modify this file at all. Please leave this as-is for historical purposes.